### ISSUE TITLE
Is there really a significant benefit in allowing InstallerType on the package-level?

### ISSUE THREAD

**Issue by donid** [microsoft/winget-pkgs#1618]
It seems to me, that having an 'InstallerType'-property on the 'Package'-level does not provide a really significant benefit.  I think it makes the development of tools that use the yaml manifests more complicated and error-prone. In my opinion the 'InstallerType'-property on the 'Installers'-level should be the only place for this information.

**Comment by MarNicGit** [comment]
I agree with you, however this issue is probably more suited for the [winget-cli](https://github.com/microsoft/winget-cli) repo since they also develop the schema over there

**Comment by donid** [comment]
@MarNicGit I always struggle with the decision where to put my issues, but I hope it will be moved, if someone of the team thinks it should be in the other repo...

**Comment by lordcheeto** [comment]
Essentially, the question is whether there exists a scenario where a piece of software would be packaged into multiple installation types, and I can think of two such scenarios:

1. Many programs are packaged into an installer, as well as standalone binaries in a ZIP file. When ZIP installation eventually receives support, this would allow you to choose which installation type you want.

2. If support is ever extended to other operating systems, they would have different installation types.

In these scenarios, the manifest needs to specify the installation type on the installer definitions themselves, instead of just having a single root type.
